### DISCUSSION TITLE
Sidekiq Pro 5.3.1 Installation Error

### DISCUSSION THREAD

**Discussion by vikramchandran** [sidekiq/sidekiq#5316] [General]
I'm currently trying to upgrade my Sidekiq Pro to version 5.3.1, and I have added this code within my Gemfile to do so:

```
source 'https://gems.contribsys.com' do
  gem 'sidekiq-pro', '5.3.1'
end
```
I have also configured my Sidekiq Pro credentials for gems.contribsys.com on bundler. However, when I run `bundle install`, I'm receiving an authentication error: `Gem::RemoteFetcher::FetchError: bad response Unauthorized 401.` 

I'm also unable to log into https://gems.contribsys.com/gems with my credentials. Just to confirm, are my Sidekiq Pro credentials outdated or is there another reason why I might be receiving this error? I'm hoping to confirm before I renew my subscription.

**Comment by mperham** [comment]
Please email support@contribsys.com with access problems. I need to know your account email or credentials so I can identify the subscription in question.

**Reply by vikramchandran** [reply]
That sounds great. I just sent an email to [support@contribsys.com](mailto:support@contribsys.com). Thank you for your help.
